go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	bitbucket.org/bertimus9/systemstat v0.5.0
 	github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
 	github.com/Microsoft/go-winio v0.6.2
-	github.com/Microsoft/hcsshim v0.12.6
+	github.com/Microsoft/hcsshim v0.12.7-0.20241003002528-bafd4c013fbd
```

Comment:
I'd suggest adding a comment on this line listing what branch the commit came from. That makes it easier if it needs to be updated.

Reply:
`This is a generated file.` :)

Reply:
it's easy to find via https://github.com/Microsoft/hcsshim/commit/bafd4c013fbd

Reply:
hopefully we'll drop it completely in O(weeks) in favor of hnslib anyway